src/resizable.directive.ts

```diff
@@ -271,6 +271,8 @@ const RESIZE_TOP_HOVER_CLASS: string = 'resize-top-hover';
 const RESIZE_BOTTOM_HOVER_CLASS: string = 'resize-bottom-hover';
 const RESIZE_GHOST_ELEMENT_CLASS: string = 'resize-ghost-element';
 
+const MAST_HAVE_ATTRIBUT: string = 'ng-reflect-validate-resize';
```

Comment:
Spelling is strange here. What is `MAST_HAVE_ATTRIBUT`? Isn't it `VALIDATE_RESIZE_ATTRIBUTE`?